automatic/chromium/update.ps1

```diff
@@ -1,47 +1,114 @@
-      "(?i)(^\s*checksum\s*type\:).*"     = "`${1} $($Latest.ChecksumType32)"
-      "(?i)(^\s*checksum32\:).*"          = "`${1} $($Latest.Checksum32)"
-      "(?i)(^\s*checksum64\:).*"          = "`${1} $($Latest.Checksum64)"
-    }
-  }
-}
-
-
-function global:au_BeforeUpdate() {
-  #Download $Latest.URL32 / $Latest.URL64 in tools directory and remove any older installers.
-  Get-RemoteFiles -Purge
-}
-
-
-function global:au_GetLatest {
-  $allVersions = Invoke-WebRequest -Uri https://api.github.com/repos/henrypp/chromium/releases -UseBasicParsing | ConvertFrom-Json
-  $allStableVersions = $allVersions | Where-Object {$_.body -match "stable"}
-  $latestStableVersionNumber = ($allStableVersions[0].tag_name.split('-') | Select-Object -First 1) -replace 'v', ''
-  $anyArchLatestStablesVersions = $allVersions  | Where-Object {$_.tag_name -match $latestStableVersionNumber}
-
-  $32LatestVersion = $anyArchLatestStablesVersions | Where-Object {$_.tag_name -match $latestStableVersionNumber -and $_.tag_name -match "win32"}
-  $32LatestSyncInstallUrl = ($32LatestVersion.assets | Where-Object name -match "-sync.exe").browser_download_url
-
-  $64LatestVersion = $anyArchLatestStablesVersions | Where-Object {$_.tag_name -match $latestStableVersionNumber -and $_.tag_name -match "win64"}
-  $64LatestSyncInstallUrl = ($64LatestVersion.assets | Where-Object name -match "-sync.exe").browser_download_url
-
-  $ChecksumType = 'sha256'
-
-  $Latest = @{
-    Version        = $latestStableVersionNumber
-    URL32          = $32LatestSyncInstallUrl
-    ChecksumType32 = $ChecksumType
-    URL64          = $64LatestSyncInstallUrl
-    ChecksumType64 = $ChecksumType
-  };
-
-  return $Latest
-}
-
-Update-Package -ChecksumFor none
+
+import-module au
+
+function global:au_SearchReplace {
+   @{
+    ".\legal\verification.txt" = @{
+        "(?i)(\s*32\-Bit Software.*)\<.*\>" = "`${1}<$($Latest.URL32)>"
+        "(?i)(\s*64\-Bit Software.*)\<.*\>" = "`${1}<$($Latest.URL64)>"
+        "(?i)(^\s*checksum\s*type\:).*"     = "`${1} $($Latest.ChecksumType32)"
+        "(?i)(^\s*checksum32\:).*"          = "`${1} $($Latest.Checksum32)"
+        "(?i)(^\s*checksum64\:).*"          = "`${1} $($Latest.Checksum64)"
+        }
+    ".\tools\chocolateyInstall.ps1" = @{
+        '(^[$]version\s*=\s*)(".*")' = "`$1""$($Latest.Version)"""
+        '(?i)(^\s*file\s*=\s*)(".*")' = "`$1""$($Latest.PackageName32)"""
+        '(?i)(^\s*file64\s*=\s*)(".*")' = "`$1""$($Latest.PackageName64)"""
+        }
+    ".\chromium.nuspec" = @{
+      "(?i)(^\s*\<title\>).*(\<\/title\>)" = "`${1}$($Latest.Title)`${2}"
+    }
+    }
+}
+
+function global:au_BeforeUpdate {
+  # Copy the original file from the chromium folder
+    Get-RemoteFiles -Purge -FileNameBase "$($Latest.PackageName)"
+}
+
+    $releases_stable = 'https://api.github.com/repos/henrypp/chromium/releases'
+    $releases_snapshots = 'https://chromium.woolyss.com/api/v2/?os=windows&bit=<bit>&out=string'
+    $PackageName = 'chromium'

```

Comment:
Don't set the `PackageName` when it isn't needed.

`PackageName` is automatically set by AU